demo/site/src/layout/footer/blocks/FooterContentBlock.tsx

```diff
@@ -23,15 +23,19 @@ export const FooterContentBlock = withPreview(
                         </TopContainer>
                         <HorizontalLine />
                         <LinkCopyrightWrapper>
-                            {linkList.blocks.length > 0 && (
-                                <LinksWrapper>
-                                    {linkList.blocks.map((block) => (
-                                        <LinkText key={block.key} as={LinkBlock} data={block.props.link} variant="p200">
-                                            {block.props.text}
-                                        </LinkText>
-                                    ))}
-                                </LinksWrapper>
-                            )}
+                            <nav>
+                                {linkList.blocks.length > 0 && (
+                                    <LinksWrapper>
+                                        {linkList.blocks.map((block) => (
+                                            <Link key={block.key}>
+                                                <LinkText key={block.key} as={LinkBlock} data={block.props.link} variant="p200">
```

Comment:
The key is only necessary for the parent:

```suggestion
                                                <LinkText as={LinkBlock} data={block.props.link} variant="p200">
```